src/devices/Arduino/README.md

```diff
@@ -0,0 +1,106 @@
+- Open the Arduino IDE
+- Go to the library manager and check that you have the "ConfigurableFirmata" library installed
+- Open "ConfigurableFirmata.ino" from the [device binding folder](./ConfigurableFirmata/ConfigurableFirmata.ino) or go to http://firmatabuilder.com/ to create your own custom firmata firmware. Make sure you have at least the features checked that you will need.
+- Upload this sketch to your Arduino. 
+
+After these steps, you can start coding with Iot.Devices.Arduino and make your Arduino do whatever you want, from blinking LEDS to your personal weather station. For usage and examples see the samples folder. Note that ConfigurableFirmata uses a default UART speed of 57600 baud. It is recommended to increase it to at least 115200, though. 
+
+When the firmware starts, the on-board-LED flashes a few times, indicating the loaded firmware version (currently 2 + 11 blinks). After that, the board will enter idle state and wait for connections. 
+
+### Advanced features
+Some of the features of this binding require extended features on the Arduino firmware. These include SPI support and DHT sensor support. These features didn't make it into the main Firmata branch yet, therefore these additional steps are required:
+- Go to C:\users\<username>\documents\arduino\libraries and delete the "ConfigurableFirmata" folder (save any work if you've changed anything there)
+- Replace it with a clone of https://github.com/pgrawehr/ConfigurableFirmata and switch to branch "develop". 
+- Make sure you have the "DHT Sensor Library" from Adafruit installed (use the library manager for that).
+- You can now enable the DHT and SPI features at the beginning of the ConfigurableFirmata.ino file. Because the new firmware will have additional features, it is recommended to use the .ino file from the examples folder of the repository. So the best start is to open the file that now lies in C:\users\<username>\documents\arduino\libraries\ConfigurableFirmata\examples\ConfigurableFirmata\ConfigurableFirmata.ino. The file has some comments at the top to enable or disable certain modules.
+- Compile and re-upload the sketch. 
+
+## Usage
+See the examples for some advanced use cases. 
+
+Basic start:
+```
+            // Portname is "COM3", "COM4" on Windows, "/dev/ttyUSB0" or similar on linux
+            using (var port = new SerialPort(portName, 115200))
+            {
+                Console.WriteLine($"Connecting to Arduino on {portName}");
+                try
+                {
+                    port.Open();
+                }
+                catch (UnauthorizedAccessException x)
+                {
+                    Console.WriteLine($"Could not open COM port: {x.Message} Possible reason: Arduino IDE connected or serial console open");
+                    return;
+                }
+
+                ArduinoBoard board = new ArduinoBoard(port.BaseStream);
```

Comment:
consider showcasing simpler constructor (portname, baudRate) so sample is shorter